paddle/fluid/framework/ir/graph.h

```diff
@@ -89,11 +93,26 @@ class Graph {
   }
 
   bool Has(const std::string &attr_name) const {
+    if (FLAGS_convert_all_blocks) {
+      if (IsMainGraph()) {
+        LOG(WARNING) << "This graph is main_graph, which doesn't have nodes or "
```

Comment:
You don't have to put a warning on it since it is a normal behavior, same for other APIs

Reply:
I have deleted the warnings.